scss/formCard.module.scss

```diff
@@ -0,0 +1,59 @@
+$light-gray: hsl(0, 0%, 85%);
+$mint-green: hsl(170, 66%, 38%);
```

Comment:
should `light-gray` and `mint-green` be more global, or are they exclusively going to be used in `formCard`?

Reply:
I felt it was a different task to modify `_variables.scss`. But you're right, `$mint-green` should be global. `$light-gray` is exclusive for the `FormCard`.